### PR TITLE
Bump design system version and update package.json

### PR DESCRIPTION
**Pull request by maciaszczykm**
## Summary
Related to ENG-270.

- Updates filter button styling in marketplace
- Bumps design system version
- Updates package.json config including switch to yarn and moving config to dedicated files

## Test Plan
Tested manually.

## Checklist
- [x] If required, I have updated the Plural documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] I have added a meaningful title and summary to convey the impact of this PR to a user.
- [x] I have added relevant labels to this PR to help with categorization for release notes.

### CODE REVIEW

File: www/package.json [hunk 1]
```diff
@@ -3,32 +3,15 @@
...
-      "not op_mini all"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
+  "engines": {
+    "node": ">=16.15.0",
+    "yarn": ">=1.22.0"
```
Comment (DavidSpek) [inline www/package.json:14]: I thought the latest version of Yarn is 3.2.0.

Comment (DavidSpek) [reply www/package.json:14]: Correction, 3.2.1. See https://yarnpkg.com/getting-started/install

Comment (DavidSpek) [reply www/package.json:14]: I'm no frontend guy so whatever makes sense haha

Comment (maciaszczykm) [reply www/package.json:14]: Done.

File: www/package.json [hunk 2]
```diff
@@ -121,8 +104,5 @@
     "eslint": "8.19.0",
     "eslint-config-pluralsh": "3.1.0",
     "postcss-import": "8.2.0"
-  },
-  "peerDependencies": {
-    "react-transition-group": ">=4.4.2"
```
Comment (maciaszczykm) [inline www/package.json:126]: @dogmar it looks that you've added it but with the changes made here I've decided to put it back with all dependencies. It looks like we are using it directly in the code. Please let me know if you would rather not change it.

**Review by michaeljguarino — COMMENTED** [review]
Is the `.yarn` folder supposed to be in this diff?

**Review by michaeljguarino — COMMENTED** [review]
We'll also need to modify the dockerfile if we're goign to be using yarn again

**Comment by maciaszczykm** [comment]
>Is the .yarn folder supposed to be in this diff?

Yes, see pt 6: https://yarnpkg.com/getting-started/migration#step-by-step

**Comment by maciaszczykm** [comment]
@michaeljguarino I've restored previous version of Dockerfile that supports yarn. Docker build passed locally. PTAL.

**Comment by DavidSpek** [comment]
> Is the .yarn folder supposed to be in this diff?

@michaeljguarino Yes, there are a couple files in there that you want to keep in the git history. I ran into this with the kubricks frontend as well.

**Review by floreks — APPROVED** [review]

**Comment by DavidSpek** [comment]
@maciaszczykm It looks like the yarn version in the Dockerfile is not berry.

```console
Step 6/13 : RUN yarn
 ---> Running in 566bcb50e9dc
yarn install v1.22.19
```

**Pull request merged**